tests/helpers/blueprint-shim.js

```diff
@@ -0,0 +1,60 @@
+var fs = require('fs-extra');
+var path = require('path');
+
+var stringifyAndNormalize = require('../../lib/utilities/stringify-and-normalize');
+var transformer = require('../../blueprints/addon/index');
+
+var processTemplate = require('../../lib/utilities/process-template');
+
+// The blueprint paths for `bower.json` and `package.json` so we're using the right things.
+var paths = {
```

Comment:
Can you get away with just doing `process.cwd()` instead of `join`ing and walking back to the root of the project?